quickstarts/microsoft.deviceupdate/deviceupdate-create-account/main.bicep

```diff
@@ -0,0 +1,15 @@
+@description('The name of the account. Max length is 24 characters. See more at https://aka.ms/iot-hub-device-update-limits')
+param accountName string = take('adu-quickstart-${uniqueString('adu-quickstart')}', 24)
+
+@description('The location of the account')
+@allowed([
+  'westus2'
+  'northeurope'
+  'southeastasia'
+])
+param location string
+
+resource accountName_resource 'Microsoft.DeviceUpdate/accounts@2020-03-01-preview' = {
```

Comment:
remove the _resource from the symbolic name - can probably just use "account" (since you'll need to disambiguate)